JavaScript_01/JavaScript_01.js

```diff
@@ -0,0 +1,75 @@
+function validate()
+{
+	let keeper1 = checkUsername();
+	let keeper2 = checkPasswd();
+	let keeper3 = checkSecondPasswd();
+	
+	if(keeper1 && keeper2 && keeper3)
+	{
+		return true;
+	}
+	else 
+	{
+		return false;
+	}
+}
+function checkUsername()
+{
+	let username = document.getElementById("Username").value;
+	let regex = RegExp('^[0-9a-zA-Z_]+$');
+	let charCount = username.length;
+	
+	if(charCount >= 3 && charCount <= 10 && regex.test(username))// || /^[a-zA-Z_]+$/.test(username))
+	{
+		//alert('Username is OK!');
+		return true;
+	}
+	else 
+	{
+		let invalidUsername = document.getElementById("invalidUsername");
+		invalidUsername.innerHTML = "Invalid username";
+		invalidUsername.style.color = "red";
+		
+		return false;
+	}
+}
+function checkPasswd()
+{	
+	let passwd = document.getElementById("Password").value;
+	let regex = RegExp('^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])[A-Za-z0-9]{6,}$');
```

Comment:
http://webtech.w3c.fmi.uni-sofia.bg/w14labs/regex.html#2 "Now they have two problems."
Would you say that this code is readable/easily understandable? Or easily modifiable by people who do not know regex? Also how error-prone is it to bugs?
Idea to consider: using multiple and simpler regex-es and applying "and" condition if all of them match